src/applications/vaos/appointment-list/components/PastAppointmentsListV2/index.jsx

```diff
@@ -0,0 +1,264 @@
+  useEffect(() => {
+    if (!showPastAppointments) {
+      history.push('/');
+    } else if (pastStatus === FETCH_STATUS.notStarted) {
+      const selectedDateRange = dateRangeOptions[pastSelectedIndex];
+      fetchPastAppointments(
+        selectedDateRange.startDate,
+        selectedDateRange.endDate,
+        pastSelectedIndex,
+      );
+    }
+  }, []);
+  useEffect(
+    () => {
+      if (pastStatus === FETCH_STATUS.succeeded && !isInitialMount) {
+        focusElement('#queryResultLabel');
+      }
+    },
+    [isInitialMount, pastStatus],
+  );
+
+  const onDateRangeChange = index => {
+    const selectedDateRange = dateRangeOptions[index];
+
+    setInitialMount(false);
+    fetchPastAppointments(
+      selectedDateRange.startDate,
+      selectedDateRange.endDate,
+      index,
+    );
+  };
+
+  let content;
+
+  if (pastStatus === FETCH_STATUS.loading) {
+    content = (
+      <div className="vads-u-margin-y--8">
+        <LoadingIndicator message="Loading your upcoming appointments..." />
+      </div>
+    );
+  } else if (
+    pastStatus === FETCH_STATUS.succeeded &&
+    appointmentsByMonth?.length > 0
+  ) {
+    content = (
+      <>
+        {appointmentsByMonth.map((monthBucket, monthIndex) => {
+          const monthDate = moment(monthBucket[0].start);
+          return (
+            <React.Fragment key={monthIndex}>
+              <h3 id={`appointment_list_${monthDate.format('YYYY-MM')}`}>
+                <span className="sr-only">Appointments in </span>
+                {monthDate.format('MMMM YYYY')}
+              </h3>
+              <ul
+                aria-labelledby={`appointment_list_${monthDate.format(
+                  'YYYY-MM',
+                )}`}
+                className="vads-u-padding-left--0"
+              >
+                {monthBucket.map((appt, index) => {
+                  const facilityId = getVAAppointmentLocationId(appt);
+                  const isCancelled = appt.status === 'cancelled';
+
+                  if (
+                    appt.vaos.appointmentType ===
+                      APPOINTMENT_TYPES.vaAppointment ||
+                    appt.vaos.appointmentType ===
+                      APPOINTMENT_TYPES.ccAppointment
+                  ) {
+                    return (
+                      <AppointmentListItem
+                        key={index}
+                        appointment={appt}
+                        facility={facilityData[facilityId]}
+                        className="vads-u-margin-bottom--2 vads-u-background-color--gray-lightest vads-u-padding--2 vads-u-margin-bottom--3"
+                      />
+                    );
+                  } else if (appt.vaos.isExpressCare) {
+                    return (
+                      <ExpressCareListItem
+                        key={index}
+                        appointment={appt}
+                        facility={facilityData[facilityId]}
+                        cancelled={isCancelled}
```

Comment:
I think I removed this cancelled prop from this component in the upcoming appts PR.

Reply:
Removed